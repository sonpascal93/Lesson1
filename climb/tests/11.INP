1234
77 5 -80 -52 -39 26 -70 -52 69 34 72 -45 -76 -27 -57 92 -12 91 -92 76 39 -26 78 43 -13 3 99 -43 -18 83 -85 18 -75 20 16 41 -65 -84 -12 -30 -60 -52 -98 73 -73 -17 -20 7 -89 -82 56 31 -24 20 -41 39 95 -88 94 69 31 -25 97 55 99 12 25 -94 -17 70 -41 -68 -26 -9 -51 -98 65 -15 -65 -90 45 -30 -7 -14 64 86 -59 44 -52 -58 -8 91 -14 83 -75 -73 18 68 -93 39 12 68 -88 -58 30 59 -9 78 32 -70 56 -30 6 10 -84 81 -98 0 55 40 7 20 -43 -22 3 -66 8 -18 -50 -56 -87 21 17 15 -69 45 20 -65 -38 -87 0 -12 -38 -98 18 -54 15 -1 86 -91 -5 97 -33 66 25 97 55 29 -95 88 -65 -32 -92 -67 -48 94 -16 52 97 -11 -49 -80 19 -55 70 82 -40 -38 77 93 36 -66 -97 98 -20 5 -57 -28 5 44 91 -30 82 0 75 89 74 73 -13 -55 -71 98 -74 33 48 -30 -32 54 -84 27 29 93 -73 -21 8 -43 -42 -59 -96 83 9 -99 -3 -5 -2 -13 56 -79 41 -72 -93 15 -44 -73 -94 -35 -35 86 -20 34 11 32 -79 54 40 -33 -93 -85 54 -38 12 43 3 70 12 27 -79 -42 -99 86 -87 -60 71 38 64 -4 -3 -60 77 85 -11 -63 -64 49 -55 79 61 -92 96 -23 -29 2 95 39 -44 -10 24 26 47 7 92 -1 -28 -87 -11 -86 93 75 19 98 85 -9 99 -28 -3 22 66 20 -12 -12 -37 41 -25 -21 84 -87 -35 -26 -26 49 -94 45 36 36 -62 55 98 -66 49 83 70 56 -55 89 -98 33 40 48 -71 -13 -50 -84 61 39 -80 -89 0 -87 -96 -74 54 -93 -86 11 -14 -51 7 84 -30 -71 -27 -47 70 59 -67 74 -36 35 67 38 14 47 85 -53 -55 -70 -85 81 -49 52 99 27 28 -36 45 -9 94 -27 30 -8 -89 44 43 99 17 -46 17 -26 -55 37 97 88 -87 -20 -75 78 63 33 80 21 21 49 33 -8 -85 76 70 -91 -28 55 -15 45 -43 77 -29 87 90 -73 -18 -27 -21 -27 -89 78 66 22 -96 3 41 -80 54 17 -87 26 -1 -21 -12 -92 -81 -62 85 -15 94 30 -6 78 34 -80 53 48 -65 -40 9 -77 -56 -1 -12 89 6 -88 -30 43 -46 -94 -68 -47 23 22 -77 -17 82 -8 -48 0 52 -83 -44 75 21 93 87 -95 38 81 15 -43 88 -72 0 71 -79 9 -20 -74 48 -59 70 -97 71 -51 -68 25 90 -67 85 88 -56 -5 72 41 -38 26 22 55 -34 47 -52 -62 -31 -19 91 6 -42 27 -54 74 64 72 83 -38 -34 -33 -90 18 -76 -95 -81 9 22 -47 -58 -71 86 -95 -72 -83 32 -53 47 -89 -80 18 59 13 4 -85 21 -26 -87 -6 78 -89 -47 3 -94 39 56 -26 -70 -38 -51 77 82 5 76 -33 -1 87 -92 95 12 -96 43 2 66 -96 87 43 94 -67 20 -19 -44 -11 -34 53 -84 21 -2 -82 38 4 25 15 -13 -3 -8 22 -55 -24 -30 21 -7 -72 17 -82 87 40 69 77 34 43 -87 53 -19 22 -63 88 -80 -74 31 66 84 17 46 -30 -30 42 -30 -32 -32 0 -6 -96 -12 -20 77 18 -14 -48 41 16 -12 -92 96 0 93 -20 91 98 19 64 -46 66 29 -7 29 36 1 -13 -6 -43 -79 41 69 -67 -56 96 -19 90 -24 40 11 -3 -89 -29 36 66 -5 -87 -21 21 20 -2 99 -87 -4 17 43 7 -75 92 -85 21 -60 19 14 -98 82 4 -51 -63 -44 84 40 62 67 22 46 -77 -76 -23 -41 81 55 53 -58 79 -28 76 71 -59 -82 -72 -98 -16 -88 24 -85 37 54 0 2 -12 50 1 99 -74 76 56 82 -8 76 74 -55 16 -8 63 -12 17 -72 -26 -45 -94 76 -27 -93 -63 -93 30 -15 49 -49 89 -14 -33 34 25 84 -61 19 98 52 83 31 -74 52 84 -84 94 92 -18 -49 -97 17 -30 53 51 -3 11 57 -60 -47 81 62 58 -17 -7 -73 78 -67 37 -57 -32 13 -42 12 70 -35 -88 70 -93 -54 -87 -65 -1 -51 -2 27 20 -74 90 44 -22 77 97 37 80 55 -28 93 96 -9 -93 -28 -90 4 -46 -40 63 92 -52 -34 25 99 -13 67 -96 68 94 -68 24 72 -54 64 -40 53 45 -70 41 86 -42 37 97 -17 74 -58 93 13 2 -53 -99 -28 -49 65 68 -45 63 -59 -28 99 -11 -55 -41 45 66 51 87 4 -22 74 15 56 -76 -98 51 83 -6 -54 40 -48 34 9 -12 -61 1 -97 20 62 61 -57 -92 60 -36 56 -62 23 10 52 86 40 43 -75 -89 -92 -79 -17 77 15 -99 -72 47 -66 83 -11 -49 -33 14 20 92 -1 76 98 -32 -87 -5 -19 31 -24 -87 95 -77 27 -4 19 -67 -70 77 -91 -79 -74 -81 -89 -50 6 -7 -84 11 -31 -12 -1 17 67 27 65 -76 29 34 -45 -72 -43 12 97 53 -77 21 2 74 19 84 -59 42 -66 26 -28 66 -24 -16 30 76 -24 -65 29 -37 42 54 -58 1 53 -28 81 -62 28 85 52 85 84 91 34 -76 -14 27 -50 -37 -17 64 -61 -61 -60 34 39 14 66 -31 6 54 -32 9 98 2 75 70 27 60 -80 -22 -42 -65 30 29 77 -67 -19 -87 -38 -48 3 -2 48 21 39 -99 50 36 31 81 -18 -11 21 96 83 91 64 -96 32 -74 -77 -24 -31 -97 -10 -31 10 15 74 -90 -85 79 14 -36 62 46 32 -73 -20 10 70 -39 36 5 -43 36 -67 60 -34 -90 -63 97 21 6 -97 -41 -36 45 93 -42 44 50 -98 83 60 -42 62 -86 95 63 -51 85 -24 10 81 50 52 -32 -49 58 -87 -88 0 -71 24 0 74 -70 5 4 7 -54 35 -30 37 62 0 13 20 43 -36 25 1 24 -11 -16 -87 76 -83 -41 -50 8 7 -43 43 -90 -84 -58 73 7 -63 87 79 38 -77 -75 -47 -2 -75 92 28 0 -44 3 -41 11 37 -99 -76 94 56 61 1 90 -60 -54 -76 -61 51 13 -26 -92 -4 58 -53 66 -14 11 