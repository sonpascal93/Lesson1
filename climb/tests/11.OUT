96
43 76 117 144 183 219 235 247 278 294 316 348 353 377 415 449 458 464 479 504 511 514 517 527 533 540 541 542 586 621 646 647 649 656 659 663 676 681 699 707 752 756 766 770 780 787 793 797 807 811 816 828 831 845 848 851 852 865 870 881 886 895 898 929 930 933 935 939 941 968 973 975 992 1002 1007 1028 1041 1045 1066 1098 1099 1102 1110 1118 1132 1136 1142 1147 1168 1194 1197 1202 1205 1215 1223 1228 